packages/database/src/migrations/20210415110921-UpdatePSSSWeeklyIndicatorsSupportSiteData-modifies-data.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const oldConfig = syndromeCode => ({
+  formula: `firstExistingValue(PSSS_${syndromeCode}_Cases, PSSS_${syndromeCode}_Daily_Cases)`,
+  aggregation: {
+    [`PSSS_${syndromeCode}_Cases`]: 'FINAL_EACH_WEEK',
+    [`PSSS_${syndromeCode}_Daily_Cases`]: ['SUM_EACH_WEEK', 'FINAL_EACH_WEEK'],
+  },
+  defaultValues: {
+    [`PSSS_${syndromeCode}_Cases`]: 'undefined',
+    [`PSSS_${syndromeCode}_Daily_Cases`]: 'undefined',
+  },
+});
+
+const siteParameters = syndromeCode => [
+  {
+    code: 'sumSiteWeeklyCases',
+    config: {
+      formula: `PSSS_${syndromeCode}_Cases`,
+      aggregation: [
+        'FINAL_EACH_WEEK',
+        {
+          type: 'SUM_PER_PERIOD_PER_ORG_GROUP',
+          config: {
+            dataSourceEntityType: 'facility',
+            aggregationEntityType: 'country',
+          },
+        },
+      ],
+    },
+    builder: 'analyticArithmetic',
+  },
+  {
+    code: 'sumSiteDailyCases',
+    config: {
+      formula: `PSSS_${syndromeCode}_Daily_Cases`,
+      aggregation: [
+        'SUM_EACH_WEEK',
+        'FINAL_EACH_WEEK',
+        {
+          type: 'SUM_PER_PERIOD_PER_ORG_GROUP',
+          config: {
+            dataSourceEntityType: 'facility',
+            aggregationEntityType: 'country',
+          },
+        },
+      ],
+    },
+    builder: 'analyticArithmetic',
+  },
+];
+
+const newConfig = syndromeCode => ({
+  formula: `firstExistingValue(PSSS_${syndromeCode}_Cases, PSSS_${syndromeCode}_Daily_Cases, sumSiteWeeklyCases, sumSiteDailyCases)`,
+  parameters: siteParameters(syndromeCode),
+  aggregation: {
+    [`PSSS_${syndromeCode}_Cases`]: 'FINAL_EACH_WEEK',
+    [`PSSS_${syndromeCode}_Daily_Cases`]: ['SUM_EACH_WEEK', 'FINAL_EACH_WEEK'],
+    sumSiteWeeklyCases: 'RAW',
+    sumSiteDailyCases: 'RAW',
+  },
+  defaultValues: {
+    [`PSSS_${syndromeCode}_Cases`]: 'undefined',
+    [`PSSS_${syndromeCode}_Daily_Cases`]: 'undefined',
+    sumSiteWeeklyCases: 'undefined',
+    sumSiteDailyCases: 'undefined',
+  },
```

Comment:
We should add support for a single string (expanded for all formula variables) for `defaultValues` at some point... We already support this for `aggregation`

Reply:
Or even a default default value?

Reply:
Very meta, haha! We probably don't want to use `default` values by default - we have to use it here to overcome the "all defined or not calculate" logic in indicators